packages/angular_devkit/core/src/jobs/README.md

```diff
@@ -0,0 +1,501 @@
+
+The job implementation should emit the `Start` and `End` events when it is starting the job logic
+itself. Only the first `Start` and `End` events will be forwarded. Any more will be filtered out.
+
+The `Queued` state is set as the job is scheduled, so there is no need to listen for the event.
+
+## `Job<OutputType>` Object
+The `Job` object that is returned when you schedule a job provides access to the job's status and 
+utilities for tracking and modifying the job. 
+
+1. `id`. A unique symbol that can be used as a Map key.
+1. `description`. The description of the job from the scheduler. See `JobDescription` object.
+1. `argument`. The argument value that was used to start the job.
+1. `input`. An `Observer` that can be used to send validated inputs to the job itself.
+1. `output`. An `Observable<OutputType>` that filters out events to get only the returned output
+    of a job.
+1. `promise`. A promise that waits for the last output of a job. Returns the last value outputted
+    (or no value if there's no last value).
+1. `state`. The current state of the job (see `LifeCycle`).
+1. `channels`. A map of side channels the user can listen to as `Observable`.
+1. `ping()`. A function that can be used to ping the job, receiving a `Promise` for when the ping
+    is answered.
+1. `stop()`. Sends a `stop` input to the job, which suggests to stop the job. The job itself can
+    choose to ignore this event.
+1. `rawInput`. The raw input `Observer<JobInput>`. This can be used to send messages to the
+    `context.rawInput` observable in the job. These are `JobInput` events. See
+    ["Communicating With Jobs"](#Communicating).
+1. `rawOutput`. The raw output `Observable<JobOutput>`. This can be used to listen to events from
+   the job. See ["Communicating With Jobs"](#Communicating).
+   
+## `JobHandlerContext<I, O>` Object
+The `JobHandlerContext<>` is passed to the job handler code in addition to its argument. The
+context contains the following members:
+
+1. `description`. The description of the job. Its name and schemas.
+1. `scheduler`. A `Scheduler<>` instance that can be used to create additional jobs.
+1. `dependencies`. A generic list of other job instances that were run as dependencies when
+    scheduling this job. Their `id` is not guaranteed to match the `id` of the `Job<>` instance 
+    itself (those `Job<>`s might just be proxies). The state of those `Job<>` is guaranteed to be
+    `JobState.Ended`, as `JobState.Errored` would have prevented this handler from running.
+1. `rawInput`. The raw input observable, complement of the `rawInput` observer from the `Job<>`.
+
+# Examples
+
+An example of a job that adds all input together and return the output value. We use a 
+simple synchronous job registry and a simple job scheduler.
+
+```typescript
+import { jobs } from '@angular-devkit/core';
+
+const add = jobs.createJobHandle<number[], number>(
+  input => input.reduce((total, curr) => total + curr, 0),
```

Comment:
Here and in other examples in the readme the job handler calls its argument `input`, but it should really be `argument`. Both Argument and Input exist as concepts and they are different.

Reply:
Fixed.